### ISSUE TITLE
Option to include instead of exclude URLs from auto save

### ISSUE THREAD

**Issue by OliverSiwers** [internetarchive/wayback-machine-webextension#955]
**Is your feature request related to a problem/ issue? Please describe.**
I think that only being able to exclude pages from auto saving is a big problem because it is easy for pages you don't want to save to end up being saved regardless. This could happen due to forgetting to add a pattern, making a mistake when adding a pattern, or simply not knowing beforehand that you didn't want to save a certain page before visiting it for the first time. 

**Describe the solution you'd like**
I think the easiest solution would be to add an option to toggle between using an include list and an exclude list.

**Describe alternatives you've considered**
Another solution could be to add something like the gitignore ! prefix that changes a regular pattern into a negating pattern. This would probably not be quite as user-friendly but would allow for more complex filtering.

**Comment by cgorringe** [comment]
Thanks for your suggestion! I will think about this some more.

**Comment by LiamSwayne** [comment]
I definitely agree with this issue. There are security concerns with constantly archiving every page a user visits. Sites that use site.com/page#decryption_key can result in leaking a secure page when auto-archive is on. Although this is bad practice, sites use it nonetheless and I've had secure documents get put in the archive this way before. Other than this feature I think the wayback machine extension covers all bases.

**Comment by DesertStars** [comment]
Agreed, this would be far more useful than the current method of excluding links.
